experiments/image-builder/overlays/ansible/roles/utilities/tasks/main.yml

```diff
@@ -13,8 +13,34 @@
     Install-RemoteAccess -VpnType RoutingOnly
     Start-Service RemoteAccess
 
-    Write-Output "Installing Calico"
-    Invoke-WebRequest https://docs.projectcalico.org/scripts/install-calico-windows.ps1 -OutFile c:\install-calico-windows.ps1
+- name: Download Calico install
+  win_get_url:
+    url: "https://docs.projectcalico.org/scripts/install-calico-windows.ps1"
+    dest: 'c:\install-calico-windows.ps1'
+  retries: 5
+  delay: 3
+
+- name: Download Kubernetes kube-proxy
+  win_get_url:
+    url: "{{ kubernetes_base_url }}/kube-proxy.exe"
+    dest: '{{ kubernetes_install_path }}\'
+  retries: 5
+  delay: 3
+
+## Install CRI and pull images
+
+- name: Download crictl
+  win_get_url:
+    url: "https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.22.0/crictl-v1.22.0-windows-amd64.tar.gz"
+    dest: '{{ programfiles.stdout | trim }}\containerd\crictl-v1.22.0-windows-amd64.tar.gz'
+  retries: 5
+  delay: 3
+
+- name: Download crictl and pull images
+  win_shell: |
+    tar xvf {{ programfiles.stdout | trim }}\containerd\crictl-v1.22.0-windows-amd64.tar.gz
+    {{ programfiles.stdout | trim }}\containerd\crictl --runtime-endpoint npipe:////./pipe/containerd-containerd pull mcr.microsoft.com/windows/nanoserver:1809
```

Comment:
there is the ability to download additional images in image builder: https://github.com/kubernetes-sigs/image-builder/blob/master/images/capi/ansible/windows/roles/load_additional_components/tasks/registry.yml

Reply:
nice will use it.